packages/salesforcedx-vscode-apex-debugger/src/messages/i18n.ts

```diff
@@ -16,8 +16,14 @@
  * If ommitted, we will assume _message.
  */
 export const messages = {
+  command_error_help_text: 'Command returned the following error',
```

Comment:
Thanks, @DatGuyJonathan . The messages look good to me. I'll mark this PR approved, for messages at least.